src/main/resources/static/assets/css/custom.css

```diff
@@ -17,3 +17,7 @@
 .container:before {
   padding-top: 15px;
 }
+
+ul > .item > li {
```

Comment:
I would make this class name more meaningful, so we could not use it my mistake in other place